### PR TITLE
added a new readme and licence in repo

### PR DESCRIPTION
**Pull request by rogdevil**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,24 @@
...
 
-This repo contains jumpstart templates, 1 directory per package.
+Welcome to the Jumpstart Templates repository! This repository contains a collection of templates for various frameworks and languages to help you kickstart your projects with ease using the Jumpstart tool.
+
+
+## Contribution Guidelines
+We welcome contributions to the Jumpstart Templates repository! Follow these guidelines to contribute:
+
+1. Fork the repository to your GitHub account.
+
```
Comment (bhupeshisanurd) [inline README.md]:
```suggestion
```
Lets get rid of this line, its causing un-necessary line break in list items, github bug 🤔 ?

File: README.md [hunk 2]
```diff
@@ -1,3 +1,24 @@
 # Jumpstart Templates
 
-This repo contains jumpstart templates, 1 directory per package.
+Welcome to the Jumpstart Templates repository! This repository contains a collection of templates for various frameworks and languages to help you kickstart your projects with ease using the Jumpstart tool.
```
Comment (bhupeshisanurd) [inline README.md]:
```suggestion
Welcome to the Jumpstart Templates! This repository contains a collection of pre-packaged templates for various frameworks and languages to help you kickstart your projects with ease using the Jumpstart CLI & VSCode extension.
```

**Review by bhupeshisanurd — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (rogdevil) [reply README.md]: daym how did that get in : | ... fixed

File: README.md (hunk 2 above)
Comment (rogdevil) [reply README.md]: done

**Review by bhupeshisanurd — APPROVED** [review]

**Pull request merged**